src/routes/+layout.svelte

```diff
@@ -195,7 +195,7 @@
         <MouseListener on:clicked={() => {}} />
         <Toasts />
         <IncomingCall />
-        <VideoPreview />
+        <!-- <VideoPreview /> This needs a rework as atm it only supports one to one calling. Maybe add ability to select source? -->
```

Comment:
Ideally this should show the video of whoever is talking at the time, I know this is more involved so no worries about this in this PR but let's get a ticket opened up so we don't forget about it